mesonbuild/coredata.py

```diff
@@ -266,8 +266,10 @@ def __init__(self, options):
         self.base_options = {}
         self.external_preprocess_args = PerMachine({}, {}, {}) # CPPFLAGS only
         self.cross_file = self.__load_cross_file(options.cross_file)
-        self.compilers = OrderedDict()
-        self.cross_compilers = OrderedDict()
+        self.compilers = PerMachine(
+            OrderedDict(),
+            OrderedDict(),
+            OrderedDict())
```

Comment:
I know this is probably longer than 78 characters if it's in one line, but unless you're planning to add default values I'd just wrap it to one line as it's not super important to read each value.

Reply:
I'll add  `# type:` which will clarify too